### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by mikelsav** [matrix-msu/Kora3#580]
Get this chrome extension and use it to correct accessibility issues. All red issues should be fixed, and fix all the yellow issues as much as possible. 
https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US

Every page on Kora will need to be checked, so take your time and be thorough. 

If you get stuck on an accessibility issue, just move on. When you have finished as much as possible, I (Nick) will come help you with the remaining problems. Keep track of how many issues remain for each page so you don't forget.
